classes/manager.php

```diff
@@ -251,9 +252,11 @@ public function capture_event(\core\event\base $event) {
             return;
         }
 
-        // Check if the user can capture this event, anti cheater method.
-        if (!$this->can_capture_event($event)) {
-            return;
+        if (get_config('enablecheatguard')) {
```

Comment:
Is this working? This looks like it would check for a site-wide setting rather than the manager's instance setting. I think this should be using `$this->get_config('enablecheatguard')`.